tests/ceph_installer/test_cephadm.py

```diff
@@ -130,7 +131,7 @@ def run(ceph_cluster: Ceph, **kwargs) -> int:
             if command == "shell":
                 cephadm.shell(**cfg)
                 continue
-
+            cfg["base_url"] = base_url
```

Comment:
I see a potential regression issue for other methods. Would it be possible to perform an E2E run?

Reply:
@psathyan i have added logs in comment 1 http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-8C3R0M/ for test_cephadm.py as well. Is there something else to test E2E?

Reply:
As discussed, could we use `self.config['base_url']`?